actions/reviews/CheckTestSuite.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# All rights reserved. The Universal Permissive License (UPL), Version 1.0 as shown at http://oss.oracle.com/licenses/upl
+# CheckTestSuite.py
+# Description: Includes the logic to perform check if test suite was included on PR
+
+from actions.utils.helpers.helper import *
+import json
+import glob
+
+
+def check_test_suite():
+    files_per_class = get_class_files()
+    for key in files_per_class:
+        class_files = files_per_class[key]
+        test_path = get_test_path(key)
+        list_of_tests = glob.glob(test_path)
```

Comment:
This part corresponds to the analysis. Decouple this on a inner function named analyze(): on the same class